specimen/observations/eml.xml

```diff
@@ -257,13 +297,13 @@ Brachiopoda</taxonRankValue>
 			<para>Naturalis Biodiversity Centre</para>
 		</funding>
 		<studyAreaDescription>
-			<descriptor citableClassificationSystem="false" name="generic">
-				<descriptorValue/>
+			<descriptor name="generic" citableClassificationSystem="false">
```

Comment:
@u43hvs kun je dit nog aanpassen: `<descriptor name="generic" citableClassificationSystem="false" />` ?

Reply:
het is mij niet duidelijk hoe ik dat kan doen behalve de hele xml opnieuw aanbieden ?

Reply:
```suggestion
			<descriptor name="generic" citableClassificationSystem="false" />
```

Reply:
Kun je hier geen nieuwe commit mee maken?

Reply:
nee ik zie niet hoe, mogelijk omdat je al gemergd hebt?
overigens zie ik ook niet precies wat voor verbetering je wilt 
want de tag descriptor wordt iets lager pas gesloten
<descriptor name="generic" citableClassificationSystem="false">
		<descriptorValue></descriptorValue>
</descriptor>

Reply:
mm, dat komt niet over, een print screen
<img width="550" alt="Schermafbeelding 2020-05-01 om 13 57 07" src="https://user-images.githubusercontent.com/59572403/80803816-b3ca2e00-8bb3-11ea-9891-116600543493.png">

Reply:
je hebt gelijk jouw verbetering moet wel maar ik krijg hem er niet in . ik zie suggested change staan, maar nergens een bevestig knop

Reply:
ik heb er nu dit van gemaakt:
```xml
<studyAreaDescription>
	<descriptor citableClassificationSystem="false" name="generic">
		<descriptorValue />
		<citation />
	</descriptor>
</studyAreaDescription>
```
en meteen op branch NBA-TEST uitgevoerd: https://github.com/naturalis/DWCAConfig/blob/NBA-TEST/specimen/observations/eml.xml

Het aanmaken van het DWCA archive gaat nu goed. Het is mij overigens toch niet helemaal duidelijk wat er niet goed was. Maar met hulp van deze beschrijving lijkt het nu dus toch opgelost: https://eml.ecoinformatics.org/schema/eml-project_xsd.html#ResearchProjectType_studyAreaDescription